isPrime.js

```diff
@@ -32,5 +37,17 @@ console.log(result) // false
 **/
 
 function isPrime(num) {
-	// Your code
+	//return false if number is 1 or less
+	if (num <== 1) {
+      return false;
+    }
+	//check to see if the remainder is = 0 when num is divided by 
+	//any number between 2 and num
+	for(i=2; i<num; i++) {
```

Comment:
you don't have to check numbers up to the input number. checking up to the square root of the input is sufficient. e. g. when you check if 18 is divisible by 3, you automatically check for 6 as well since 3x6 = 18. so if you check up to sqrt(18) + 1 ~ 5, you are done. this would be cut down the running time of the code.